### DISCUSSION TITLE
登录微软账号后 IE 提示是否记住密码

### DISCUSSION THREAD

**Discussion by lsyanling** [Meloong-Git/PCL#170] [提问] [answered]
![image](https://user-images.githubusercontent.com/65443331/193164920-92671900-2e01-4675-9ade-2855c8644c4c.png)

我想不明白为啥会是 IE 啊

操作系统：Windows11 22H2正式版

**Comment by Big-Cake-jpg** [comment, marked as answer]
PCL2 的微软账户登陆调用的是 WPF 中的 `
WebBrowser` 控件，它使用的是 IE。

如果要解决，需要等 PCL2 更改登录流程，使用 Azure Active Directory 应用获取信息。#122
